handler/src/main/java/io/netty/handler/ssl/JdkApplicationProtocolNegotiator.java

```diff
@@ -32,6 +32,29 @@
          * Abstract factory pattern for wrapping an {@link SSLEngine} object. This is useful for NPN/APLN support.
          *
          * @param engine The engine to wrap.
+         * @param applicationNegotiator The application level protocol negotiator
+         * @param isServer <ul>
+         * <li>{@code true} if the engine is for server side of connections</li>
+         * <li>{@code false} if the engine is for client side of connections</li>
+         * </ul>
+         * @return The resulting wrapped engine. This may just be {@code engine}.
+         */
+        SSLEngine wrapSslEngine(
```

Comment:
We can but it's also exposed as the return type of a method on an interface. Let me just pull it in for now and we can do a followup if needed